src/components/Jumpstart.tsx

```diff
@@ -24,8 +51,123 @@ const Template: React.FC = () => {
         <meta name="description" content="Write a description for SEO here" />
       </Helmet>
-      <Box my={4}>
-        <Paper className={classes.example} />
+      <Hero
+        subtitle="Our brand new full-stack development program for first time developers."
+        title="Jumpstart"
+        SvgNode={JumpstartSvg}
+      />
+
+      <Grid
+        container
+        justify="space-between"
+        alignItems="flex-start"
+        direction="row"
+        className={classes.section}
+        spacing={6}
+      >
+        <Grid item md={5}>
+          <Typography variant="h4">What is Jumpstart?</Typography>
+          <Typography variant="body1" className={classes.subtext}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris
+            vitae eros laoreet nisl ornare dignissim at sit amet leo. Integer
+            convallis malesuada elit, fringilla venenatis eros facilisis ut. Nam
+            ut accumsan mauris. Aenean at odio dolor. In ultricies ipsum risus,
+            nec porttitor turpis ornare in.
+          </Typography>
+        </Grid>
+        <Grid item md={5}>
+          <Typography variant="h4">What will I learn?</Typography>
+          <Typography variant="body1" className={classes.subtext}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris
+            vitae eros laoreet nisl ornare dignissim at sit amet leo. Integer
+            convallis malesuada elit, fringilla venenatis eros facilisis ut. Nam
+            ut accumsan mauris. Aenean at odio dolor. In ultricies ipsum risus,
+            nec porttitor turpis ornare in.
+          </Typography>
+        </Grid>
+        <Grid item md={5}>
+          <Typography variant="h4">Can I participate?</Typography>
+          <Typography variant="body1" className={classes.subtext}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris
+            vitae eros laoreet nisl ornare dignissim at sit amet leo. Integer
+            convallis malesuada elit, fringilla venenatis eros facilisis ut. Nam
+            ut accumsan mauris. Aenean at odio dolor. In ultricies ipsum risus,
+            nec porttitor turpis ornare in.
+          </Typography>
+        </Grid>
+        <Grid item md={5}>
+          <Typography variant="h4">Something something</Typography>
+          <Typography variant="body1" className={classes.subtext}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris
+            vitae eros laoreet nisl ornare dignissim at sit amet leo. Integer
+            convallis malesuada elit, fringilla venenatis eros facilisis ut. Nam
+            ut accumsan mauris. Aenean at odio dolor. In ultricies ipsum risus,
+            nec porttitor turpis ornare in.
+          </Typography>
+        </Grid>
+      </Grid>
+
+      <Box className={classes.flexSection}>
+        <Button variant="contained" color="primary" className={classes.button}>
+          Sign Me Up!
```

Comment:
Instead of setting font size of the button in classes.button, use a Typography component to wrap the Sign Me Up!

<Button ...>
    <Typography variant="h5???">Sign Me Up! </Typography>
</Button>

This is better bc its responsive and the size is consistent across the site

Reply:
ah that's it